proto/uber/cadence/api/v1/workflow.proto

```diff
@@ -45,10 +45,12 @@ message WorkflowExecutionInfo {
   SearchAttributes search_attributes = 10;
   ResetPoints auto_reset_points = 11;
   string task_list = 12;
+  TaskListKind task_list_kind = 17;
```

Comment:
nit: How about introducing `TaskList task_list_metadata` and marking `string task_list` deprecated?

Reply:
Switched to `task_list_info`